commands/narrator/vt.js

```diff
@@ -25,17 +28,57 @@ module.exports = {
             if (player) {
                 droppy.options.push({ label: `${i}`, value: `votefor-${i}`, description: `${player.user.tag}` })
             }
         }
         let row = { type: 1, components: [droppy] }
         let m = await voteChat.send({ content: `Timer set to ${ms(timer)} <@&${aliveRole.id}>`, components: [row] })
+
+        // loop through each dead player
+        for (const player of deadPlayers) {
+            let guy = db.get(`player_${player}`) // get the player - Object
+            if (guy.status === "Alive") continue // if the player is alive, don't do anything and check for the next player
+
+            // revive the player
+            if (guy.ritualRevive !== gamephase + 1) continue // if the phase isn't over yet, don't do anything and check for the next player
+
+            db.set(`player_${guy.id}.status`, "Alive") // set the status of the player to Alive
+            let member = await message.guild.members.fetch(guy.id) // get the discord member
+            let memberRoles = member.roles.cache
+                .map((r) => (r.name === "Dead" ? ["892046206698680390", "892046205780131891"] : r.id))
+                .join(",")
+                .split(",") // get the roles, and replace the dead role with alive
+            await dayChat.send(`${getEmoji("ritualist_revive", client)} The Ritualist revived **${players.indexOf(guy.id) + 1} ${guy.username} (${getEmoji(guy.role?.toLowerCase()?.replace(/\s/g, "_"), client)} ${guy.role})**`) // sends a message in day chat
+            await member.roles.set(memberRoles)
+
+            for (const seerapp of seerappprentices) {
+                if (db.get(`player_${seerapp}`).originalPlayer !== guy.id) continue
+
+                let allRoles = db.get(`player_${seerapp}.allRoles`)
+                allRoles.pop()
+                db.set(`player_${seerapp}.allRoles`, allRoles)
+                db.delete(`player_${seerapp}.originalRole`)
+                db.set(`player_${seerapp}.role`, "Seer Apprentice")
+
+                let channel = guild.channels.cache.get(db.get(`player_${seerapp}`)?.channel)
+                channel?.send(`The **${guy.role}** has been revived so you have become a **Seer Apprentice** again.`)
+                channel?.send(`${guild.roles.cache.find((r) => r.name === "Alive")}`)
+                channel?.edit({ name: `priv-seer-apprentice` })
+            }
+        }
+
+        // alchemist
+
+        const alchemists = require("./night/kills/alchemists.js")
+        const zombies = require("./night/kills/zombies.js")
+        await alchemists(client)
+        await zombies(client)
+
         db.set(`commandEnabled`, `yes`)
         db.add(`gamePhase`, 1)
-        if(message?.author) message.channel.send(`Setting the vote time for ${ms(timer)}`)
+        if (message?.author) message.channel.send(`Setting the vote time for ${ms(timer)}`)
```

Comment:
```suggestion
        message.channel.send(`Setting the vote time for ${ms(timer)}`)
```
if there is no message.author, the command won't be executed